libs/remix-ui/debugger-ui/src/lib/debugger-ui.tsx

```diff
@@ -36,6 +36,28 @@ export const DebuggerUI = (props: DebuggerUIProps) => {
     sourceLocationStatus: ''
   })
 
+  const panelsRef = useRef<HTMLDivElement>(null)
+  const debuggerTopRef = useRef(null)
+
+  const handleResize = () => {
+    if (panelsRef.current && debuggerTopRef.current) {
+      panelsRef.current.style.height = (window.innerHeight - debuggerTopRef.current.clientHeight) - debuggerTopRef.current.offsetTop - 7 +'px'
+    }
+  }
+
+  useEffect(() => {
```

Comment:
It is not advice-able to use a useEffect without a dependency array, it can crash the app. Please provide a dependency array for this `useEffect`.